src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,30 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    user,
+    completed,
+    title,
+  } = todo;
+
+  return (
+    <article className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': completed,
+      },
+    )}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+      {user && <UserInfo user={user} />}
```

Comment:
I totally share the temptation to make it in one line, but you have to overcome it 😄 
readable code > short code
```suggestion
      {user && (
        <UserInfo user={user} />
      )}
```